Automattic/sensei
# Update learning mode template preview

Resolves #6917 

## Proposed Changes
* Update learning mode template previews

## Testing Instructions

1. Reset the browser cache.
2. Go to Sensei LMS -> Settings -> Appearance
3. Make sure you see updated previews (thumbs and full).

## References
Default:
![Pi7_Resizedefault](https://github.com/Automattic/sensei/assets/329356/fbeef435-99a5-4318-8f68-d5e1eab9fc15)

Modern:
![Pi7_Resizemodern](https://github.com/Automattic/sensei/assets/329356/64e456de-5b49-4757-93cf-397a3dc60c98)

Video:
![Pi7_Resizevideo](https://github.com/Automattic/sensei/assets/329356/1e9d668e-bf9d-4239-9d3f-624c0d5c0571)

Large Video:
![Pi7_Resizevideo-large](https://github.com/Automattic/sensei/assets/329356/2eca7b57-0141-43b0-90d6-daf0c3939261)

## Pre-Merge Checklist
Both the PR author and reviewer are responsible for ensuring the checklist is completed. -->
